### DISCUSSION TITLE
Inquiry for Translation of MIMIC Dataset into six languages

### DISCUSSION THREAD

**Discussion by unnir** [MIT-LCP/mimic-code#1472] [MIMIC-General] [answered]
Hi All, 

As a member of the [DataTools4Heart](https://cordis.europa.eu/project/id/101057849) project, we are seeking to translate an existing medical dataset into six European languages: Spanish, Italian, Romanian, Czech, Swedish, and Dutch using professional translators with guarantees. 

Given the popularity of the MIMIC dataset in the community, it is our top priority for translation. Can you please provide information on who we should contact to discuss the possibility of translating the MIMIC dataset?

Kind regards,
Vadim

**Comment by tompollard** [comment, marked as answer]
Thanks @unnir, sounds like an interesting project and we would be happy to chat! Please drop us an email at mimic-support@physionet.org

**Comment by unnir** [comment]
Thank you for the quick response and contact information!
